fili-core/src/main/java/com/yahoo/bard/webservice/async/ResponseException.java

```diff
@@ -88,14 +152,22 @@ private static String buildMessage(
             String description,
             int statusCode,
             DruidQuery<?> druidQuery,
-            Throwable cause
+            Throwable cause,
+            ObjectWriter objectWriter
     ) {
+        String druidQueryString;
+        try {
+            druidQueryString = objectWriter.writeValueAsString(druidQuery);
+        } catch (JsonProcessingException e) {
+            druidQueryString = druidQuery.toString();
+            LOG.warn(String.format("Failed to properly serialize druid query %s", druidQueryString), e);
```

Comment:
All, makes sense. With that in mind, you can actually just in-line `druidQuery` in the `LOG` call, and let the logging framework call `toString` if it decides it needs to actually log it.